docs/dietpi_sbc.md

```diff
@@ -9,7 +9,23 @@ title: DietPi Supported SBCs
 
 The Raspberry Pi has sold more than 31 million units since it launched in 2012. Any Raspberry Pi can serve as a secondary computer for email and web surfing. The real fun starts when you use your Pi for projects, which can range from robots to retro arcade machines to home media servers, security cameras and so on.
 
-You can install DietPi by following the step by step [Starting guide](../user-guide_install_sbc/).
+You can install DietPi by following the step by step [Starting guide](../user-guide_installation/).
+
+DietPi runs very well on latest model **Raspberry Pi 4 Model B** (launched in 2019 / 2020)
+
+<img src="../assets/images/raspberry-pi-4-labelled.png" title="Raspberry Pi model 4B" width="500" />
```

Comment:
While this works to resize the image, there is a way via Markdown as well:
```
![raspberry-pi-4-model-b](assets/images/raspberry-pi-4-labelled.png){: style="width:500px"}
```
should work. But probably there is no benefit.

Reply:
Thanks for the hint. The benefit of the Markdown variant is the reference at the beginning of the line (for adding linked references to the figure). This I did not add.